src/core/qgsvectorlayerutils.h

```diff
@@ -319,6 +319,16 @@ class CORE_EXPORT QgsVectorLayerUtils
      * \since QGIS 3.12
      */
     static QString getFeatureDisplayString( const QgsVectorLayer *layer, const QgsFeature &feature );
+
+    /**
+     * \returns true if at least one feature of the \a fids on \a layer is connected as parent in at
+     * least one composition relation of the \a project or contains joins, where cascade delete is set.
+     * The \a context is used to get more details.
```

Comment:
```suggestion
     * Details about cascading effects will be written to \a context.
```
How does that sound?

Reply:
Pretty good :-)